pandas/core/array_algos/quantile.py

```diff
@@ -181,5 +181,8 @@ def _quantile_ea_fallback(
     assert res.ndim == 2
     assert res.shape[0] == 1
     res = res[0]
-    out = type(values)._from_sequence(res, dtype=values.dtype)
+    try:
+        out = type(values)._from_sequence(res, dtype=values.dtype)
+    except TypeError:
+        out = np.atleast_2d(np.asarray(res, dtype=np.float64))
```

Comment:
in the Int64 case we probably want Float64 result right?

Reply:
should be the same behavior as 1.2.5?

Reply:
Below is from the bug description:

>**Expected Output**
If we want to go with the 1.2.5 output, the expected output would be to not error and result in a Series with dtype object. It could also make sense for the resulting series to have float64 dtype.

I have used this, and returned `float64`

Reply:
umm, why don't we just make `dtype=np.float64` always

Reply:
tried that in https://github.com/pandas-dev/pandas/pull/42936/commits/c1857bf34c10f5ea093957e8281cdc8d85e3109e, but there are multiple tests to ensure we get the same extensionDtype in pandas/tests/frame/methods/test_quantile.py::TestQuantileExtensionDtype

Reply:
also breaks the world if we have an all-pd.NA column

Reply:
add a comment here on why this is needed.  why do we need the reshape? e.g. we dont' have it on the original.

Reply:
For creating a block from non-EA we need 2d array.
https://github.com/pandas-dev/pandas/blob/8a4f29a66cb41b00e477b2bf09e05832a3967017/pandas/core/internals/blocks.py#L1316

Added a comment pointing back to the issue.
Alternatively, we can also return `Float64` as @jbrockmendel  suggested above. Because 1.2.5 returned `object` (and not `np.float64`).